src/renderer/components/uielements/assets/assetInput/AssetInput.tsx

```diff
@@ -22,6 +22,7 @@ type Props = {
   onFocus?: FP.Lazy<void>
   disabled?: boolean
   className?: string
+  maxInfo?: boolean
```

Comment:
Why not just using a `maxInfoText: string` for props instead of a boolean. With it you can pass any text you want to have for the tooltip. `AssetInput` might not be used for `Swap` only.